app/templates/content/index.hbs

```diff
@@ -44,6 +44,9 @@
             </div>
         </div> {{!END CONTAINER DIV}}
     </div> {{!END HEADER ROW}}
+    {{#if isAdmin}}
```

Comment:
No, moderators don't have special permissions on the Preprints side of things; moderators can only see the status of submissions on the Reviews app. Does that go against what you would expect?